flatpak-node-generator.py

```diff
@@ -0,0 +1,1651 @@
+#!/usr/bin/env python3
```

Comment:
No need to bundle this script, unless you've edited it.

Reply:
Well... that was a bad mistake... Corrected.